src/frontend/src/binder/expr/function/mod.rs

```diff
@@ -321,6 +321,15 @@ impl Binder {
                 self.ensure_table_function_allowed()?;
                 return Ok(TableFunction::new_file_scan(args)?.into());
             }
+            // `table_scan` table function
+            if func_name.eq_ignore_ascii_case("postgres_query") {
```

Comment:
FYI case folding is already handled by `real_value()` above, and it is actually incorrect to ignore case when doubled quoted.

But it is the fault of this PR as it is just copying a previous bug above.